packages/firmware/src/gps.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include <Adafruit_FONA.h>
+
+typedef struct {
+    float latitude;
+    float longitude;
+    float speed_kph;
+    float heading;
+    float speed_mph;
+    float altitude;
```

Comment:
Are you guys going to want to store all these values on the back-end? Or just latitude / longitude?

Reply:
Nevermind, I see the migration is already there.

Reply:
What migration are you referring to?

Reply:
The one for `locations` in the server package.

Reply:
https://github.com/shiftdsm/Galavanting-Gnome/blob/master/packages/server/db/migrations/20190423190306_create_locations.js

Reply:
That one just has `lat` and `lon`, so it'll need to change (unless thats what you meant by saying you see it).